scripts/docs-deploy/snapshot-deploy.ts

```diff
@@ -0,0 +1,67 @@
+import {cloneDocsRepositoryForMajor} from './clone-docs-repo';
+import {performDefaultSnapshotBuild} from '../build-packages-dist';
+import {buildDocsContentPackage} from '../build-docs-content';
+import {installBuiltPackagesInRepo} from './install-built-packages';
+import {DeploymentInfo, deployToSite, isProductionDeployment} from './deploy-to-site';
+import {installDepsAndBuildDocsSite} from './utils';
+import {runMonitoringTests} from './monitoring/index';
+
+export type DeploymentConfig = {
+  /** Optional hook running before building the docs-app for deployment. */
```

Comment:
It might be worth noting that this is after the assets are built.

Reply:
good idea 👍